backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/VdsManager.java

```diff
@@ -1247,10 +1257,42 @@ public void vmsMonitoringInitFinished() {
         if (!isInitialized()) {
             log.info("VMs initialization finished for Host: '{}:{}'", cachedVds.getName(), cachedVds.getId());
             resourceManager.handleVmsFinishedInitOnVds(cachedVds.getId());
+            updateCpuPinning(vmDao.getMonitoredVmsRunningByVds(cachedVds.getId()));
             setInitialized(true);
         }
     }
 
+    private void updateCpuPinning(List<VM> vms) {
```

Comment:
do I understand correctly that this is only used when engine start? if so, please rename or document that